commands/apps_test.go

```diff
@@ -45,3 +47,137 @@ func TestCommandsAppsCreateAttemptsToValidateStackOnError(t *testing.T) {
 	assert.Error(err)
 	assert.Regexp("bad request: unable to find stack", err)
 }
+
+// Looks like nested functions are not yet supported by the compiler
+func OverwriteFile(loc string, data string) (err error) {
+	err = os.Remove(loc)
+	if err != nil {
+		log.Println("[ERROR] unable to remove files, perhaps they do not exist?")
+	}
+	f, err := os.Create(loc)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	_, err = f.WriteString(data)
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func TestInitNodejsBasicAppEncounteringPossibleFailureStates(t *testing.T) {
+	assert := assert.New(t)
+
+	// Setup
+	var testCases = []struct {
+		servConf string
+		pkgJson  string
+		isFatal  bool
+	}{
+		{`location / {
```

Comment:
Let's use `helperLoadBytes` to clean up these tests. 

That way we can put the test data in `commands/testdata/apps/init/` and move data out of these tests.

Reply:
Here's [an example of this approach](https://github.com/section/sectionctl/blob/583716bfef5195e3d7ae3b54031cf2a7be81c89f/commands/apps_test.go#L27) inside this test file itself.

Reply:
Will clean up the test cases to look like this:

``` golang
var testCases = []struct {
	servConf string
	pkgJson  string
	isFatal  bool
}{
	{ string(helperLoadBytes(t, "apps/init.good.server.conf`")),  string(helperLoadBytes(t, "apps/init.bad.package.json`")), false },
	{ string(helperLoadBytes(t, "apps/init.bad.server.conf`")),  string(helperLoadBytes(t, "apps/init.bad.package.json`")), true },
}
```